hardware/raspberrypi/booteeprom.md

```diff
@@ -98,22 +98,23 @@ sudo rpi-eeprom-update -d -f new.bin
+At power on, the BCM2711 ROM looks for a file called `recovery.bin` in the root directory of the boot partition on the sd-card. If a valid `recovery.bin` is found then the ROM executes this instead of the SPI EEPROM image. This mechanism ensures that the bootloader SPI EEPROM can always be reset to a valid image with factory default settings.
 
 #### EEPROM update files
 | Filename | Purpose |
 |----------|---------|
 | recovery.bin | VideoCore EEPROM recovery executable |
 | pieeprom.upd | Bootloader EEPROM image |
-| pieeprom.bin | Bootloader EEPROM image (same as pieeprom.upd) |
+| pieeprom.bin | Bootloader EEPROM image - same as pieeprom.upd but with changes recovery.bin behaviour |
 | pieeprom.sig | The sha256 checksum of bootloader image (pieeprom.upd/pieeprom.bin) |
-| vl805.bin | The VLI805 USB firmware EEPROM image. Pi 4 revision 1.4 and newer do not have a dedicated SPI EEPROM for the VL805 USB controller |
+| vl805.bin | The VLI805 USB firmware EEPROM image - ignored on 1.4 board revision which does not have a dedicated VLI EEPROM|
 | vl805.sig | The sha256 checksum of vl805.bin |
 
-* If the bootloader update image is called `pieeprom.upd` then `recovery.bin` renames itself to `recovery.000` and resets the CPU. Since `recovery.bin` is no longer present the ROM loads the newly bootloader from SPI EEPROM and the OS is booted as normal.
-* The `.sig` files should just contain the sha256 checksum (in hex) of the corresponding image file. Other fields should may be added in the future.
-* If the bootloader update image is called `pieeprom.bin` the `recovery.bin` will stop after the update has completed. On success the HDMI output will be green and the green activity LED is flashed rapidly. Otherwise, the HDMI output will be read and an error code will be displayed via the activity LED.
-* The BCM2711 ROM does not support loading of recovery.bin from USB mass storage or TFTP. Instead, newer versions of the bootloader support a self-update mechanism where the SPI bootloader is able to reflash the SPI EEPROM itself. See `ENABLE_SELF_UPDATE` on the (bootloader configuration)[bcm2711_bootloader_config.md] page.
+* If the bootloader update image is called `pieeprom.upd` then `recovery.bin` renames itself to `recovery.000` and resets the CPU. Since `recovery.bin` is no longer present the ROM loads the newly updated bootloader from SPI EEPROM and the OS is booted as normal.
+* If the bootloader update image is called `pieeprom.bin` the `recovery.bin` will stop after the update has completed. On success the HDMI output will be green and the green activity LED is flashed rapidly. Otherwise, the HDMI output will be red and an error code will be displayed via the activity LED.
```

Comment:
Probably makes sense to line "error code" to https://github.com/raspberrypi/documentation/blob/master/configuration/led_blink_warnings.md ?

Reply:
added. shouldn't there be some sort of wiki style autolinking for docs?